chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -177,11 +180,9 @@ impl Actor for PeerManagerActor {
         let state = self.state.clone();
         let clock = self.clock.clone();
         ctx.spawn(wrap_future(async move {
-            let mut interval =
-                tokio::time::interval(UPDATE_ROUTING_TABLE_INTERVAL.try_into().unwrap());
-            interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
+            let mut interval = time::Interval::new(clock.now(), UPDATE_ROUTING_TABLE_INTERVAL);
```

Comment:
```suggestion
            let mut interval = clock.new_interval(UPDATE_ROUTING_TABLE_INTERVAL);
```

would read a bit better I think